package.json

```diff
@@ -7,7 +7,7 @@
     "dev": "vite",
     "build": "tsc && vite build",
     "preview": "vite preview",
-    "test": "vitest"
+    "test": "vitest --run"
```

Comment:
:pen: This flag ends test runner after running the tests instead of waiting to re-run tests upon a saved code change.